src/components/common/item-selector/dropdown-list.js

```diff
@@ -37,9 +37,10 @@ export const ListItem = ({value, displayOption = defaultDisplay}) => (
 );
 
 const DropdownListWrapper = styled.div`
-  background-color: ${props => props.theme.dropdownListBgd};
+  background-color: ${props =>
+    props.light ? props.theme.dropdownListBgdLT : props.theme.dropdownListBgd};
   border-top: 1px solid ${props => props.theme.dropdownListBorderTop};
-  ${props => props.theme.dropdownList};
+  ${props => (props.light ? props.theme.dropdownListLT : props.theme.dropdownList)};
```

Comment:
shouldn't the theme be in charge of providing the correct color without having kepler checking for __light__?

Reply:
well yes and no, this is the dropdown inside a modal, and all models are having the light theme. so we need a way to render the dropdown differently